lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/TransactionPayload.kt

```diff
@@ -1,9 +1,13 @@
 package com.swmansion.starknet.data.types.transactions
 
+import com.swmansion.starknet.data.serializers.JsonRpcTransactionPayloadPolymorphicSerializer
 import com.swmansion.starknet.data.types.*
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
+@Serializable(with = JsonRpcTransactionPayloadPolymorphicSerializer::class)
+sealed class TransactionPayload
```

Comment:
So you managed to get the `@Serializable(with = ...)` syntax working. 👍 I remember I had some issues with that previously.